### PR TITLE
Contact form functionality

### PR DESCRIPTION
**Pull request by mrvanagas**
Added contact form. Form validation and error handling is handled by Formik:
image
We can see, that if email in invalid, it will show an invalid email error, checking by Regex. We can also see, that comment input field is getting an error, since it is empty. Once all fields have some sort input, only then the form will allow to submit.

Once submitted, the inputs are stored in localStorage:
image

### CODE REVIEW

File: client/src/Components/ContactForm.jsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+}
+
+//here is Form validation, checking if fields have info in them and email validation regex
+const validate = values => {
+  let errors = {}
+
+  if(!values.name) {
+      errors.name = 'Required'
+  } else if (values.name.length < 3) {
+    errors.name = 'Must be atleast 3 characters'
```
Comment (JuliaShavyrova) [inline client/src/Components/ContactForm.jsx]: please fix typos in messages

File: client/src/Components/ContactForm.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      errors.comments = 'Required'
+  } else if (values.comments.length < 20) {
+    errors.comments = 'Must be atleast 20 characters'
+  }
+
+  return errors
+}
+
+const ContactForm = () => {
+  const formik = useFormik({
```
Comment (JuliaShavyrova) [inline client/src/Components/ContactForm.jsx]: variable can be destructured

Comment (mrvanagas) [reply client/src/Components/ContactForm.jsx]: Destructured variable, also refactored the code, to make it more readable, by using `getFieldProps` method

**Comment by JuliaShavyrova** [comment]
contact form component looks good to me, but i cannot merge this PR, because it contains code from PR #209 that's not approved

**Comment by mrvanagas** [comment]
Resolved conflicts, should be ready for merge

File: client/src/Components/ContactForm.jsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    errors.comments = 'Required';
+  } else if (values.comments.length < 20) {
+    errors.comments = 'Must be atleast 20 characters';
+  }
+
+  return errors;
+};
+
+const ContactForm = () => {
+  const {handleSubmit, errors, touched, getFieldProps} = useFormik({
```
Comment (JuliaShavyrova) [inline client/src/Components/ContactForm.jsx]: please don't forget to check formatting before commiting/creating PRs

Comment (mrvanagas) [reply client/src/Components/ContactForm.jsx]: Formatting should be done for this branch.

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**